modules/recommended-etcd-practices.adoc

```diff
@@ -64,8 +57,12 @@ $ sudo docker run --volume /var/lib/etcd:/var/lib/etcd:Z quay.io/openshift-scale
 ----
 --
 
-The output reports whether the disk is fast enough to host etcd by comparing the 99th percentile of the fsync metric captured from the run to see if it is less than 10 ms.
+The output reports whether the disk is fast enough to host etcd by comparing the 99th percentile of the fsync metric captured from the run to see if it is less than 20 ms. These are some of the most important etcd metrics that could be potentially affected by I/O performance:
```

Comment:
I agree that `etcd_disk_backend_commit_duration` is also an important metric, nevertheless, since all etcd writes (Including raft heartbeats) are first performed to the WAL which is tracked by `etcd_disk_wal_fsync_duration_seconds_bucket`, I think adding specific thresholds to the metric `etcd_disk_backend_commit_duration` won't add much value.